release notes/v0.40.0.md

```diff
@@ -0,0 +1,249 @@
+k6 v0.40.0 is here! :tada: 
+#TODO 
+[Roadmap and future plans section](#roadmap-and-future-plans).
+
+## New Features and notable changes!
+
+### Breaking change!: Main module/script no longer taints the global scope [#2571](https://github.com/grafana/k6/pull/2571)
+
+As part of the ESM changes it was found out that anything defined in the scope of the main module also was accessible globally. This was because it was directly evaluated in the global scope.
+
+This has now been remedied and is no longer the case. This is a *breaking* change, but given that the whole point of modules, CommonJS or ESM, is to separate modules - so this is obviously a bug.
+
+We already have reports by people who have knowingly or not used it, but it seems fairly rare and contained to one or two usages in a script. The recommended way is to if you need something globally to set it explicitly on the global object `globalThis`.
+
+### Experimental modules [#2630](https://github.com/grafana/k6/pull/2630) and [#2656](https://github.com/grafana/k6/pull/2656)
```

Comment:
```suggestion
### Experimental Redis [#2630](https://github.com/grafana/k6/pull/2630) and and WebSocket API [#2656](https://github.com/grafana/k6/pull/2656) modules
```

Reply:
They were added in the same PR (the first one) teh second is an update of all of them